<launch>
    <!-- Control arguments -->
    <arg name="show_rviz" default="true" />

    <!-- Visualization -->
    <node if="$(arg show_rviz)" name="rviz" pkg="rviz" type="rviz" args="-d $(find imarker_detector_ros)/config/rviz_single_vision.rviz" output="screen" />

    <!-- Main Node -->
    <node name="iMarker_algorithms" pkg="imarker_detector_ros" type="runner_rs.py" output="screen">
        <rosparam command="load" file="$(find imarker_detector_ros)/config/cfg_single_rs.yaml" />
    </node>

</launch>